src/server.ts

```diff
@@ -55,16 +55,29 @@ export class Server {
 
   // Connect mongoose to our database
   private async connectDB() {
+    const rpcServer: RPC = new RPC(config.rpc_port);
```

Comment:
Also, this is not just a rpc server it's a grpc server of the file service ,  so maybe change it to fileserver or any other indicative name as I already suggested...

Reply:
better?

Reply:
No🙈

Reply:
so... do you have a suggestion?

Reply:
If you give your variable name like `fileServer` then why to call the class `FILE`? 
either name both of them `file` or as I prefer call the class and the variable `FileServer`

@omrishtam what is your opinion?

Reply:
That’s what I thought of.